tests/rptest/services/redpanda.py

```diff
@@ -455,6 +457,9 @@ def __init__(self,
         self._enable_sr = enable_sr
         self._security = security
 
+        if node_ready_timeout_s != None:
+            self.NODE_READY_TIMEOUT_SEC = node_ready_timeout_s
```

Comment:
All-caps is reserved for constants -- suggest changing `NODE_READY_TIMEOUT_SEC` to `DEFAULT_NODE_READY_TIMEOUT_SEC`, then this code would be like

```
if node_ready_timeout_s is None:
    node_ready_timeout_s = DEFAULT_NODE_READY_TIMEOUT_S
self.node_ready_timeout_s = node_ready_timeout_s
```

Reply:
Good point, updated